README.md

```diff
@@ -96,7 +96,6 @@ Shown below is the full list of attributes which you can specify within your Flu
   - `true`: Override the default existing Flutter launcher icon for the platform specified
   - `false`: Ignore making launcher icons for this platform
   - `icon/path/here.png`: This will generate a new launcher icons for the platform with the name you specify, without removing the old default existing Flutter launcher icon.
-- `image_path`: The location of the icon image file which you want to use as the app launcher icon
```

Comment:
This is still used

Please undo this removal

Reply:
@MarkOSullivan94 I think this is a duplicate of the line that exists in the global section, this confuses me. Also, this one is not displayed in the IOS section. Is that right, or is ios missing this line? If not, I will revert the change. Thanks!

![image](https://github.com/user-attachments/assets/42bc1c93-dbf8-4dfa-9def-3adb087ace1e)